pages/common/mmdc.md

```diff
@@ -0,0 +1,16 @@
+# mermaid-cli
+
+> A command-line interface (CLI) for mermaid. It takes a mermaid definition file as input and generates svg/png/pdf file as output.
+> More information: <https://mermaid-js.github.io/mermaid/>.
+
+- Convert file to format speficied in file extension (svg|png|pdf):
```

Comment:
Perhaps this will read better?
```suggestion
- Convert file to the specified format (automatically determined from the file extension. svg, png, and pdf are supported):
```

Reply:
sounds good to me, would it be smart to change `output.svg` in the example to something more descriptive?
i.e
`mmdc --input {{input.mmd}} --output {{output.svg|output.png|output.pdf}}`
instead of
`mmdc --input {{input.mmd}} --output {{output.svg}}`